resources/virtual-machines/create-vm.js

```diff
@@ -169,14 +133,11 @@ const createNIC = async (subnetInfo, publicIPInfo) => {
       },
     ],
   };
-  console.log("\n5.Creating Network Interface: " + networkInterfaceName);
-  return await networkClient.networkInterfaces.createOrUpdate(
-    resourceGroupName,
-    networkInterfaceName,
-    nicParameters
-  );
-};
-const findVMImage = async () => {
+  const resCreate = await networkClient.networkInterfaces.createOrUpdate(resourceGroupName,networkInterfaceName,nicParameters);

```

Comment:
The same here. 
I remember network package will not return the result in their polling could you add a networkInterfaces.get right after the beginCreateOrUpdateAndWait to show the full resource info ?